en/docs/try-out/account-information-service-flow.md

```diff
@@ -8,66 +8,41 @@ Service (AIS).
-               issuer_dn = "EMAILADDRESS=malshani@wso2.com, CN=OB, OU=OB, O=WSO2, L=COL, ST=WP, C=LK"
-               ```
-        4. Restart the servers.
-        5. Download the following certificates and keys, and use them for testing purposes.
-            - Use the [transport private key](../../assets/attachments/transport-certs/obtransport.key) and
-              [transport public certificate](../../assets/attachments/transport-certs/obtransport.pem) for Transport
-              layer security testing purposes.
-            - Use the [signing certificate](../../assets/attachments/signing-certs/obsigning.pem) and
-              [signing private keys](../../assets/attachments/signing-certs/obsigning.key) for signing purposes.
-
-    ``` tab='Format'
-    
-    {
-    "alg": "<The algorithm used for signing.>",
-    "kid": "<The thumbprint of the certificate.>",
-    "typ": "JWT"
-    }
-    {
-    "iss": "<This is the issuer of the token. For example, client ID of your application>",
-    "sub": "<This is the subject identifier of the issuer. For example, client ID of your application>",
-    "exp": <This is the epoch time of the token expiration date/time>,
-    "iat": <This is the epoch time of the token issuance date/time>,
-    "jti": "<This is an incremental unique value>",
-    "aud": "<This is the audience that the ID token is intended for. For example, https://<IS_HOST>:9446/oauth2/token>"
-    }
-   
-    <signature: The client assertion must be signed using the private key of the application certificate.>
+1. Run the following cURL command in a command prompt to generate the access token. Update the placeholders with relevant values.
+
+   ??? note "Use the sample certificates for testing purposes. Click here to see how it is done..."
+       1. Download the [cert.pem](../../assets/attachments/cert.pem) and upload it to the client trust stores as follows:
+       - The client trust stores for the Identity Server and API Manager are located in the following locations:
+       - `<APIM_HOME>/repository/resources/security/client-truststore.jks`
+       - `<IS_HOME>/repository/resources/security/client-truststore.jks`
+       2. Use the following commands to add the certificate to the client trust store:
+       ```shell
+       keytool -import -alias cert -file <PATH_TO_CERT.PEM> -keystore client-truststore.jks -storepass wso2carbon
+       ```
+       3. Open the `<APIM_HOME>/repository/conf/deployment.toml` file and update the following configurations:
+       - Add `SHA1withRSA` as a supported signature algorithm:
+       ```
+       supported_signature_algorithms = ["SHA256withRSA", "SHA512withRSA", "SHA1withRSA"]
+       ```
+       - Update the following tag according to the sample:
+       ```
+       [[open_banking.gateway.certificate_management.certificate.revocation.excluded]]
+       issuer_dn = "EMAILADDRESS=malshani@wso2.com, CN=OB, OU=OB, O=WSO2, L=COL, ST=WP, C=LK"
```

Comment:
Shall we use a placeholder instead of the personal mail address here?

Reply:
Added a seperate point with the placeholders for the viewers to refer in general. Kept the above as it is mentioning to configure like this if the test certificates are used. Because the test certificates contain the above email address in it. Fixed in https://github.com/wso2/docs-ob-toolkit-berlin/pull/83/commits/032d27a14d369ed185a66165e900ede55c908bbe